pages/agent/v3/cli_annotate.md.erb

```diff
@@ -1,13 +1,19 @@
 # buildkite-agent annotate
 
-The Buildkite Agent’s `annotate` command allows you to add additional information to Buildkite build pages using CommonMark Markdown. 
+The Buildkite Agent’s `annotate` command allows you to add additional information to Buildkite build pages using CommonMark Markdown.
 
 <%= image "overview.png", alt: "Screenshot of annotations with test reports" %>
 
 <%= toc %>
 
 ## Creating an annotation
 
+The `buildkite-agent` cli's `annotate` command creates an annotation associated with the current build.
```

Comment:
I don't think we refer to `cli` anywhere for the agent, and might be confused with https://github.com/buildkite/cli — perhaps just this?

```suggestion
The `buildkite-agent annotate` command creates an annotation associated with the current build.
```